iroha/src/config.rs

```diff
@@ -96,6 +107,11 @@ impl ConfigurationBuilder {
                 .kura_block_store_path
                 .unwrap_or_else(|| DEFAULT_KURA_BLOCK_STORE_PATH.to_string()),
             trusted_peers: self.trusted_peers,
+            max_faulty_peers: self
+                .max_faulty_peers
+                .unwrap_or_else(|| DEFAULT_MAX_FAULTY_PEERS.to_string())
+                .parse()
+                .unwrap_or_else(|_| panic!("Failed to parse {}", MAX_FAULTY_PEERS)),
```

Comment:
Use expect

Reply:
Was planning to do it like that `.expect(format!("Failed to parse {}", MAX_FAULTY_PEERS).as_str())` but clippy suggested `unwrap_or_else`. Strangely it doesn't suggest it now, so I will change it to expect then.

Reply:
Yep, CI with clippy actually fails with `expect` and suggests usage of `unwrap_or_else`, so I will leave it as it was.